certbot-nginx/certbot_nginx/configurator.py

```diff
@@ -69,7 +69,8 @@ class NginxConfigurator(common.Installer):
 
     @classmethod
     def add_parser_arguments(cls, add):
-        add("server-root", default=constants.CLI_DEFAULTS["server_root"],
+        default_server_root = _determine_default_server_root()
+        add("server-root", default=default_server_root,
```

Comment:
Up to you if you want to fix this, but by actually setting the default to `default_server_root`, if you use the nginx plugin with `CERTBOT_DOCS` set and without using `--nginx-server-root`, it tries to use the value "/etc/nginx or /usr/local/etc/nginx" as the server root with `nginx`.

If we wanted to fix this, we can take advantage of some magic added to `certbot/cli.py` and do something like https://github.com/certbot/certbot/commit/3cb02c08394d98c0fd65993c06e5c2e96d139777.

Reply:
I like it